minor: missing script prepublish

<!--
  Please note that this template is not optional.
  Please fill out _ALL_ fields, or your issue may be closed as "invalid."
  Please do not delete this template.
  Please ask questions on StackOverflow or Gitter (https://gitter.im/webpack/webpack).
  General questions, how-to questions, and support requests will be closed.
-->

- Operating System: Fedora 31
- Node Version: 12.16.1
- NPM Version: 6.13.14
- webpack Version: 4.43.0
- webpack-dev-server Version: 3.11.0
- Browser: Google Chrome 81.0.4044.122 


<!--
  Please place an x, no spaces, in all [ ] that apply
  Please note that we are NOT accepting FEATURE requests at this time.
-->

- [ ] This is a **bug**
- [x] This is a **modification** request

The `CONTRIBUTING.md` file mentions about running `npm run prepublish` [here](https://github.com/webpack/webpack-dev-server/blob/master/CONTRIBUTING.md#setting-up-a-local-copy) to recompile all the files. However, there is no such [script](https://github.com/webpack/webpack-dev-server/blob/master/package.json#L16).

### Expected Behavior
Files in `client/` are recompiled.

### Actual Behavior

`npm ERR! missing script: prepublish`

### Suggestion
Either change the script to be run to `npm run prepare` or add a `prepublish` script. I'd like to send a PR for this.

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -31,7 +31,7 @@ Run the relevant [examples](https://github.com/webpack/webpack-dev-server/tree/m
 
 Once it is done, you can modify any file locally. In the `examples/` directory you'll find a lot of examples with instructions on how to run it. This can be very handy when testing if your code works.
 
-If you are modifying a file in the `client/` directory, be sure to run `npm run prepublish` after it. This will recompile the files.
+If you are modifying a file in the `client/` directory, be sure to run `npm run build:client` after it. This will recompile the files.
 
 ## Testing a Pull Request
 

